deployment/eclipse-kuksa/README.md

```diff
@@ -0,0 +1,69 @@
+<!--
+******************************************************************************
+Copyright (c) 2019 Bosch Software Innovations GmbH [and others].
+All rights reserved. This program and the accompanying materials
+are made available under the terms of the Eclipse Public License v2.0
+which accompanies this distribution, and is available at
+https://www.eclipse.org/org/documents/epl-2.0/index.php
+***************************************************************************** 
+-->
+
+# Eclipse Kuksa Deployment
+
+---
+
+## Build Eclipse Kuksa Appstore
+
+* Script:
+  * `build_kuksa_appstore.sh`
+* Purpose:
+  * Build a Docker image for the Eclipse Kuksa Appstore and push it to a Docker registry
+* Options:
+  * `DOCKER_REGISTRY_SECRET`: Name of the secret to access to custom Docker registry. **Note that the secret is created during the build and deploy process within the namespace `extension`.**
+  * `DOCKER_REGISTRY_SERVER`: Address of the Docker registry server, preferrably running on Microsoft Azure.
```

Comment:
Why it is Azure preferred? Maybe we change this to something like "e.g. running on Microsoft Azrue")